dlt/destinations/path_utils.py

```diff
@@ -92,7 +92,7 @@ def prepare_datetime_params(
     if load_package_timestamp:
         current_timestamp = ensure_pendulum_datetime(load_package_timestamp)
         params["load_package_timestamp"] = str(int(current_timestamp.timestamp()))
-        params["load_package_timestamp_ms"] = current_timestamp.format("SSS")
+        params["load_package_timestamp_ms"] = str(int(current_timestamp.timestamp() * 1000))
```

Comment:
imho we have a place for these kind of function (for example timestamp to seconds or timestamp to ms) which is dlt.common.time. it should go there as util function (the line above too) and there should be some basic tests that only test this function.

Reply:
I like it, I think this will be helpful for us.

Reply:
@sh-rp I extracted two helper functions.